src/main/java/io/github/cottonmc/resources/type/ResourceType.java

```diff
@@ -1,17 +1,17 @@
-package io.github.cottonmc.resources;
+package io.github.cottonmc.resources.type;
 
 import net.minecraft.block.Block;
 import net.minecraft.item.Item;
 
 import javax.annotation.Nullable;
 
 public interface ResourceType {
-    /** Gets the base resource name. For example, "copper". */
+    /** Gets the base resources name. For example, "copper". */
```

Comment:
looks like you did a search-and-replace of `resource` to `resources`, and it *really* scrambled this diff with bad grammar.